source/components/utilities/utdebug.c

```diff
@@ -161,6 +161,8 @@
 
 #ifdef ACPI_DEBUG_OUTPUT
 
+static DEFINE_SPINLOCK(AcpiUtdebug_lock);
```

Comment:
in ln 164 there is a typo and it needs to be _Lock which is what is used everywhere else below this definition.